### ISSUE TITLE
Punch list for 0.5.0

### ISSUE THREAD

**Issue by jbednar** [bokeh/datashader#325]
Various tasks we hope to do before releasing 0.5.0:

* [x] Verify line example after off-by-one changes [JC]
* [x] Fix dashboard to use census.parq [JC]
* [x] Make a new census.parq: [JC]
   - [x] Use snappy compression
   - [x] Use single, not double, precision
   - [x] Change to use northing,easting, not metersnorth,meterswest
   - [x] Upload to S3 bucket
* [x] Fixes/improvements for `census.*` notebooks: [JC]
   - [x] Update to use new census.parq, with recommended options as in #313 (fixed in #348)
* [ ] Update all examples to use holoviews where appropriate (JB; can be done after release)
* [x] Test and fix all notebook examples (legend, hover likely have issues) (JC,JB)
* [x] Add conda package for newer pip-only python-snappy version (JC) (conda-forge pkgs already exist)
* [x] Update conda packages to get higher performance: dask>0.14.1, fastparquet>0.0.5 (waiting on next release), numba (after new PR) (JC to verify that these get installed; may need to specify numba channel) (fixed in #349)
* [x] Update docs to include performance recommendations from #313 and #129.
